src/main/java/com/cyber/domain/Review.java

```diff
@@ -0,0 +1,41 @@
+package com.cyber.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "Reviews")
+public class Review {
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private long id;
```

Comment:
Use wrappers Long, Byte instead of primitives

Reply:
Already done [https://github.com/SportResourceDevelopers/CyberServer/pull/1/commits/f9539d61bb1629749f1729706a9d11a52bc9cfe2](url)